plugins/CorePluginsAdmin/templates/macros.twig

```diff
@@ -101,6 +101,21 @@
                 });
             });
         });
+        
+        function boxes_selected(){
```

Comment:
In general we don't use snake case in matomo, we use camel case (so this function name would be `boxesSelected`).